cli/mfg.c

```diff
@@ -44,6 +44,14 @@
 #include <stdio.h>
 #include <string.h>
 
+static const struct argconfig_choice secure_state_choices[] = {
+	{"INITIALIZED_UNSECURED", SWITCHTEC_INITIALIZED_UNSECURED,
+	 	"Unsecured"},
```

Comment:
Indent issue and leading white space. There're other indent issues in this patch.